### ISSUE TITLE
know-your-http-well docset cannot be added

### ISSUE THREAD

**Issue by despairblue** [zealdocs/zeal#301]
Adding `https://raw.githubusercontent.com/for-GET/know-your-http-well/master/docset/know-your-http-well.xml` leads to `“zeal” terminated by signal SIGSEGV (Address boundary error)`.

**Comment by trollixx** [comment]
I've tried with the latest code base. No crashes, but the  [archive](https://raw.github.com/for-GET/know-your-http-well/master/docset/know-your-http-well.docset.tgz) is really different. It looks like it contains a two docsets inside and a lot of other stuff. Also, I've noticed, that index database has lower case name `docset.dsidx` instead of `docSet.dsidx`, and for some reason Zeal cannot parse `Info.plist`. I'll try to figure out later, how to handle such archives.

Thanks for reporting!
